Lib/lib2to3/Grammar.txt

```diff
@@ -110,7 +110,7 @@ atom: ('(' [yield_expr|testlist_gexp] ')' |
        '[' [listmaker] ']' |
        '{' [dictsetmaker] '}' |
        '`' testlist1 '`' |
-       NAME | NUMBER | STRING+ | '.' '.' '.')
```

Comment:
I'm pretty sure this is required to parse Python 2, where something like `x[.                                     ..]` is a perfectly valid way of writing `x[...]`.

Reply:
Sigh, you're right. I doubt anybody actually does this in practice though.

But yeah, for completion we'd have to retain atoms with three dots. Technically not a loss in functionality but definitely a loss in convenience for the programmer.